examples/1-getting-started/src/app/Home.tsx

```diff
@@ -32,28 +66,28 @@ export const Home: FC = () => {
     return (
       <ul>
         {res.data.todos.map((todo: ITodo) => (
-          <Todo key={todo.id} {...todo} />
+          <Todo
+            deleteTodo={() => executeDeleteTodoMutation({ id: todo.id })}
+            toggleTodo={() => executeToggleTodoMutation({ id: todo.id })}
+            key={todo.id}
+            {...todo}
+            loading={toggleTodoMutation.fetching}
+            disabled={
+              toggleTodoMutation.fetching || deleteTodoMutation.fetching
+            }
+          />
         ))}
       </ul>
     );
   }, [res]);
```

Comment:
isn't the linter complaining here because toggleTodoMutation is missing and so on?